core/src/test/scala/org/broadinstitute/dsde/rawls/googleProject/GoogleProjectRepositorySpec.scala

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.dsde.rawls.googleProject
+
+import org.broadinstitute.dsde.rawls.dataaccess.slick.TestDriverComponent
+import org.broadinstitute.dsde.rawls.model.{RawlsBillingProject, RawlsGoogleProject}
+import org.scalatest.flatspec.AnyFlatSpec
+
+import java.util.UUID
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+class GoogleProjectRepositorySpec extends AnyFlatSpec with TestDriverComponent {
+
+  behavior of "createGoogleProject"
+
+  def makeGoogleProject(billingProject: RawlsBillingProject) = RawlsGoogleProject(
+    UUID.randomUUID().toString,
+    billingProject.billingAccount.map(_.toString),
+    Some("fake message"),
+    billingProject.projectName
+  )
+
+  it should "create a google project record" in withDefaultTestDatabase {
+    val repo = new GoogleProjectRepository(slickDataSource)
+    val googleProject = makeGoogleProject(testData.testProject1)
+
+    val result = Await.result(repo.createGoogleProject(googleProject), Duration.Inf)
+
+    assertResult(googleProject) {
+      result
+    }
+  }
```

Comment:
here - and in the Component - do we want to allow setting the `message` at insert time? I thought the message was here to hold any errors we hit when changing a billing account or otherwise operating on a pre-existing project. If so, we'd only set the message on updates, not inserts. 

I also don't feel strongly about this at all, it's definitely not a blocker in any way as long as end users can't set/erase their own messages

Reply:
I'll leave it out of the test, but I think since it's not included in the API I wouldn't worry about putting in any safeguards.